src/extensions/replay/sessionrecording-utils.ts

```diff
@@ -69,6 +69,7 @@ export declare type recordOptions<T> = {
     maskTextSelector?: string
     maskAllInputs?: boolean
     maskInputOptions?: MaskInputOptions
+    dangerouslyCapturePasswordInputs?: boolean
```

Comment:
tbh I think this is overkill... I don't see how the issue is the existing naming. The issue is that the default was to not mask them whereas now the default is always to mask them and you have to explicitly mark it as false? 

I think we should instead leave it as it was but explicitly check the object and mark it as "true" if it is anything other than explicitly "false"

Reply:
Fair point. I've fixed things up to ensure a `password` value is always present. If not set we ensure the default of _true_ is used